recipes/portable-file-dialogs/all/conanfile.py

```diff
@@ -0,0 +1,29 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class PortableFileDialogsConan(ConanFile):
+    license = "WTFPL"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/samhocevar/portable-file-dialogs"
+    description = "Portable GUI dialogs library, C++11, single-header"
+    topics = ("conan", "gui", "dialogs")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy("portable-file-dialogs.h", dst="include", src=self._source_subfolder)
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+
+    def package_id(self):
+        self.info.header_only()
+
+    def package_info(self):
+        self.cpp_info.libdirs = []
```

Comment:
needed? does the hook complain about this?

Reply:
I had a warning from conan itself

Reply:
```
portable-file-dialogs/0.1.0: WARN: Lib folder doesn't exist, can't collect libraries: C:\Users\SSE4\.conan\data\portable-file-dialogs\0.1.0\_\_\package\5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9\lib
[HOOK - conan-center.py] post_package_info(): WARN: [LIBRARY DOES NOT EXIST (KB-H054)] Component portable-file-dialogs::portable-file-dialogs libdir "lib" does not contain any declared library
```